stablehlo/tests/interpret_clamp.mlir

```diff
@@ -0,0 +1,74 @@
+// RUN: stablehlo-interpreter --interpret -split-input-file %s | FileCheck %s
+
+// CHECK-LABEL: Evaluated results of function: clamp_op_test_si4
```

Comment:
Per [testing guidelines](https://github.com/openxla/stablehlo/blob/main/docs/reference.md), you should use the highest precision for all types.

Reply:
Got it, thanks!